backend/codesirius/problems/migrations/0006_alter_sampletest_options_sampletest_created_at_and_more.py

```
@@ -0,0 +1,172 @@
+            options={
+                "verbose_name": "Sample Test",
+                "verbose_name_plural": "Sample Tests",
+            },
+        ),
+        migrations.AddField(
+            model_name="sampletest",
+            name="created_at",
+            field=models.DateTimeField(
+                auto_now_add=True,
+                default=datetime.datetime(
+                    2025, 3, 16, 9, 23, 22, 20341, tzinfo=datetime.timezone.utc
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)